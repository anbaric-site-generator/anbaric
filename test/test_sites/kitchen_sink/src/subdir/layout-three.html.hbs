<h1>layout 3</h1>
{{{ content }}}
