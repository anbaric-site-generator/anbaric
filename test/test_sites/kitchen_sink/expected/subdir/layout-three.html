<html>
  <head></head>
  <body>
    <h1>layout 3</h1>
  </body>
</html>